OrbitLinuxTracing/PerfEventOpen.cpp

```diff
@@ -109,17 +109,6 @@ int uprobes_retaddr_event_open(const char* module, uint64_t function_offset,
   return generic_event_open(&pe, pid, cpu);
 }
 
-int uprobes_stack_event_open(const char* module, uint64_t function_offset,
```

Comment:
since I cannot leave a review on commit message:

Please fix it it should be "Remove unused functions from ...."

Reply:
Done.